package.json

```diff
@@ -36,6 +36,8 @@
         "@dnd-kit/core": "^6.0.8",
         "@dnd-kit/sortable": "^7.0.2",
         "@hookform/resolvers": "^3.3.4",
+        "@lifesg/react-design-system": "^2.6.0-canary.4",
+        "@lifesg/web-frontend-engine": "^1.10.3",
```

Comment:
design system and frontend engine are meant to be under `peerDependencies`, i.e. consumer is expected to have them installed in their repo rather than bundling them in this library, this keeps the library small

along with `peerDependencies`, we will install these 2 packages as `devDependencies` for dev use

Reply:
the corresponding changes need to be done on the package-lock as well
the easiest way is to discard your changes to the package-lock.json and install both libraries as devdependencies again